src/main/scala/ai/privado/languageEngine/javascript/tagger/source/IdentifierTagger.scala

```diff
@@ -124,8 +124,10 @@ class IdentifierTagger(cpg: Cpg, ruleCache: RuleCache, taggerCache: TaggerCache)
             // typeDeclVal: models.py:<module>.Profile
             val impactedObjects = cpg.identifier
               .filter(n => typeDeclVal.endsWith(n.typeFullName))
-//              _.astSiblings failing in some cases to removing this filter for now.
-//              .whereNot(_.astSiblings.isCall.name("import"))
+              .filter(n => {
+                if (n.contains("astSiblings")) { n.astSiblings.isCall.name("import|require").isEmpty }
```

Comment:
Do we need this check?

Reply:
Yes, else scan for some repos is breaking.

Reply:
Interesting, can you try with `n.start.astSiblings` that should work

Reply:
tried  not working.